Makefile

```diff
@@ -1,25 +1,13 @@
-
-
 build:
 	$(MAKE) -C apt-hook build
 
-deps:
-	./tools/read-dependencies -v 3 --system-pkg-names --test-distro
```

Comment:
I use this target to quickly setup a clean vm that can be used as a dev/test environment, maybe we can migrate some of your multipass goodness in here to make that setup easy:
deps:
        k-build-deps -t "apt-get --no-install-recommends --yes" -i ./debian/control